e2e/fee_middleware_test.go

```diff
@@ -1,29 +1,37 @@
 package e2e
 
 import (
-	"os"
+	"context"
 	"testing"
 
+	"github.com/strangelove-ventures/ibctest/ibc"
 	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/ibc-go/v4/e2e/testsuite"
 )
 
 func TestFeeMiddlewareTestSuite(t *testing.T) {
 	suite.Run(t, new(FeeMiddlewareTestSuite))
 }
 
 type FeeMiddlewareTestSuite struct {
-	suite.Suite
+	testsuite.E2ETestSuite
 }
 
 func (s *FeeMiddlewareTestSuite) TestPlaceholder() {
```

Comment:
the place holder test now creates chains in a container and starts a relayer (no assertions are made yet!)